src/scripts/scroll_to_bottom.js

```diff
@@ -1,15 +1,17 @@
-import { keyToClasses, descendantSelector } from '../util/css_map.js';
+import { keyToClasses, keyToCss, descendantSelector } from '../util/css_map.js';
 import { translate } from '../util/language_data.js';
 import { pageModifications } from '../util/mutations.js';
 
 let knightRiderLoaderSelector;
+let loaderSelector;
 let scrollToBottomButton;
 let scrollToBottomIcon;
 let active = false;
 
 const scrollToBottom = () => {
   window.scrollTo({ top: document.documentElement.scrollHeight });
-  if (document.querySelector(knightRiderLoaderSelector) === null) {
+  if ([...document.querySelectorAll(knightRiderLoaderSelector)]
+    .some(element => element.matches(loaderSelector)) === false) {
```

Comment:
Multi-line conditions are a little rough to read. Let's see...

```suggestion
  const knightRiderLoaders = [...document.querySelectorAll(knightRiderLoaderSelector)];
  const shouldKeepScrolling = knightRiderLoaders.some(element => element.matches(loaderSelector));
  if (!shouldKeepScrolling) {
```

Reply:
👍